models/engine/file_storage.py

```diff
@@ -79,3 +79,22 @@ def close(self):
         Method calls reload method to deserialize JSON file to objects
         '''
         self.reload()
+
+
+    def get(self, cls, id):
+        '''
+        Retrieves one object of cls.id
+        Returns None upon failure
+        '''
+        key = cls + '.' + id
+        dictionary = self.all(cls)
+        return dictionary[key] if key in dictionary else None
```

Comment:
great code! simple and easy to understand. could have more descriptive variables